contracts/contracts/coordination/Coordinator.sol

```diff
@@ -127,8 +127,7 @@ contract Coordinator is AccessControlDefaultAdminRules {
     }
 
     function setProviderPublicKey(bytes calldata publicKey) external {
-        // TODO: Verify public key length
-        require(publicKey.length == 48, "Invalid public key length");
+        require(publicKey.length == 104, "Invalid public key length");
```

Comment:
We have G2 element size in here:
https://github.com/nucypher/nucypher-contracts/blob/da35b9d9f13ddebd7ce7bbbc3fbc7d5fb0d95411/contracts/contracts/lib/BLS12381.sol#L11
I guess this 104 comes from the usual extra 8 bytes for length from Rust. What do you think about using the `G2Point` struct from our `BLS12381` lib?

Reply:
Sounds good, I should update serialization in `ferveo-py` bindings then

Reply:
Ferveo changes: https://github.com/nucypher/ferveo/pull/136